### PR TITLE
Included subclasses of Lakes, Rivers and Drainagebasins.

### PR DESCRIPTION
**Pull request by Govind9**

### CODE REVIEW

File: tests/test_waterwheel.py [hunk 1]
```diff
@@ -47,6 +47,11 @@ def test_common_names(self):
         self.assertEqual(str(doc.ents[0]), 'Is')
         self.assertEqual(str(doc.ents[1]), 'is')
         self.assertEqual(str(doc.ents[2]), 'IS')
+    
+    def test_known_bugs(self):
```
Comment (lintool) [inline tests/test_waterwheel.py]: Maybe rename this `test_issue20` to line up with #20 ?

Comment (lintool) [reply tests/test_waterwheel.py]: While you're at it, can you add in "great slave lake" (lowercase as a test)?

**Comment by lintool** [comment]
@Govind9 minor comments, otherwise looks good.

File: tests/test_waterwheel.py (hunk 1 above)
Comment (Govind9) [reply tests/test_waterwheel.py]: Ref #2 discussed rule that we want to treat river/lakes as proper names eg Great Slave Lake and ignore anything which is either fully capital or fully lowercase. Should we apply this rule only to the common words or keep it as it is i.e. apply to any match?

> While you're at it, can you add in "great slave lake" (lowercase as a test)?

Comment (lintool) [reply tests/test_waterwheel.py]: I think if an entity ends in "lake" or "river" etc. it should still be recognized even if it consists of common words?

Comment (Govind9) [reply tests/test_waterwheel.py]: Made the changes to rules and added test cases.

**Pull request merged**